db.go

```diff
@@ -63,8 +63,8 @@ func (db *DB) Authenticate(token string) (interface{}, error) {
 
 // --------------------------------------------------
 
-func (db *DB) Live(table string) (string, error) {
-	id, err := db.send("live", table)
+func (db *DB) Live(table string, updateType LiveQueryUpdateType) (string, error) {
```

Comment:
Could we remove `LiveQueryUpdateType` and replace it with just `bool` please? It looks like that is how the protocol is implemented.
https://surrealdb.com/docs/integration/websocket/text#live